src/twisted/mail/relay.py

```diff
@@ -45,8 +45,8 @@ def exists(self, user):
         """
         if self.willRelay(user.dest, user.protocol):
             # The most cursor form of verification of the addresses
-            orig = filter(None, str(user.orig).split('@', 1))
-            dest = filter(None, str(user.dest).split('@', 1))
+            orig = [c for c in str(user.orig).split('@', 1) if c]
+            dest = [c for c in str(user.dest).split('@', 1) if c]
```

Comment:
Isn't this splitting-and-counting equivalent to `'@' in user.orig and '@' in user.dest`, or am I missing something?